example/string_generator.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+"""
+CherryPy String Generator Service.
+
+This is the example random string generator service, slightly
+modified for swagger to not use session support.
+"""
+import random
+import string
+import os
+import cherrypy
+
+STRING_FILE = 'string.txt'
+
+
+@cherrypy.expose
+class StringGeneratorWebService(object):
+    """String Generator Class."""
+
+    @cherrypy.tools.json_out()
+    # pylint: disable=no-self-use
```

Comment:
I cannot imagine this. Is there any particular TravisCI build with app failing with this error?